m2c/arch_mips.py

```diff
@@ -1182,6 +1182,13 @@ def eval_fn(s: NodeState, a: InstrArgs) -> None:
             outputs = [Register("hi"), Register("lo")]
             if args[0] != Register("zero"):
                 outputs.append(args[0])
+               
+            inputs = [args[1], args[2]]
+            if mnemonic in ("msub", "msubu"):
+                inputs.append(Register("lo"))
+            outputs = [Register("hi"), Register("lo")]
+            if args[0] != Register("zero"):
+                outputs.append(args[0])
```

Comment:
Just extend the `mnemonic in ("madd", "maddu")` check above.

(overwriting the `inputs`/`outputs` that were calculated above would be wrong!)